go/libkb/chain_link.go

```diff
@@ -762,3 +1021,13 @@ func (c *ChainLink) Copy() ChainLink {
 func (c ChainLink) LinkID() LinkID {
 	return c.id
 }
+
+func (c ChainLink) NeedsSignature() bool {
+	if !c.IsStubbed() {
```

Comment:
This function is only called when `IsStubbed` is true, right? Do we need this check?